vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectDetailsComponent.java

```diff
@@ -145,26 +149,11 @@ private void initFormFields() {
 
   private Span createPlaceHolderSpan() {
     Span placeholderSpan = new Span("None");
-    placeholderSpan.addClassName(TextColor.SECONDARY);
+    placeholderSpan.addClassName("placeholder");
     return placeholderSpan;
   }
 
-  private void setComponentStyles() {
-    titleToggleComponent.getInputComponent().setSizeFull();
-    projectObjectiveToggleComponent.getInputComponent().setWidthFull();
-    experimentalDesignToggleComponent.getInputComponent().setWidthFull();
-    titleToggleComponent.setWidthFull();
-    projectObjectiveToggleComponent.setWidthFull();
-    experimentalDesignToggleComponent.setWidthFull();
-    formLayout.setClassName("create-project-form");
-    projectManagerToggleComponent.getInputComponent().getStyle()
-        .set("--vaadin-combo-box-overlay-width", "16em");
-    projectManagerToggleComponent.getInputComponent().getStyle()
-        .set("--vaadin-combo-box-width", "16em");
-    principalInvestigatorToggleComponent.getInputComponent().getStyle()
-        .set("--vaadin-combo-box-overlay-width", "16em");
-    responsiblePersonToggleComponent.getInputComponent().getStyle()
-        .set("--vaadin-combo-box-width", "16em");
+  private void setFormLayoutComponentStyles() {
```

Comment:
Where did the form layout class name go?

Reply:
There is no need for a dedicated class name. 
I noticed that the create-project-form css class was deleted long ago and replaced the individual width assignment in the java class via css in the page-area.css:

.page-area.project-details-component .display-edit-component {
  width: 100%
}

.page-area.project-details-component .display-edit-component .edit-component {
  width: 100%;
}

.page-area.project-details-component .display-edit-component .display-component {
  width: 100%;
  font-size: var(--lumo-font-size-s);
}